### DISCUSSION TITLE
Enable SSC/TK 2.4 change to allow Vashj/Kael to be fought before the other bosses are defeated.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#7039] [Suggestions]
I propose that the 2.4 change to allow Vashj/Kael to be defeated before other bosses be turned on. Specifically, Vashj should be available early now, and Kael should be made accessible early with Hyjal's release. Reason I say this is because this was a change made with Sunwell and the change was accurate to 3.3.5, and other changes inaccurate for 3.3.5 have been removed (e.g. attunement) with the next raid release.

**Comment by Cooties-cc** [comment]
For people with these raids on farm, I can see this is a reasonable alternative now that mythic is pretty much confirmed dead. Since there are some items that last quite some time in there, but are restricted by more or less time gating, it could be nice to pick and choose the most time efficient bosses to kill. Currently with the 2 raids, you can more or less clear both in around 2.5 - 3 hours with a decent comp/gear. However with the advent of Hyjal, and future raids, cutting this time down is essential if trying to only do 1 raid night. Even with de-tuning of raids, it's still quite bothersome just on a time scale. Could be cut down a little more if some trash wasn't so insanely scaled up. 

While fixing and maintaining issues is still key, I don't think going by the patch change to have more options is dire. You could chart a pros & cons list, but often the pros would win by over 50%. I'm not sure if this is big enough to garnish a poll vote for the server, but since most of the "active" players are raiders, more than likely it'd win. Leaves it up to devs for choice on the matter, perhaps also getting feedback from the "guild reps".

**Comment by IskrenPlashkov** [comment]
I for one fully approve this idea. Vashj and Kael had their time under the sun, its tier 6 time now. Make them available for the few people who still need items from them. It is also in line with the general idea of nerfing older content.

**Comment by UndoUreche** [comment]
I have absolutely nothing to say/add. I'm just here to show some support. This is a great idea.

**Comment by naritafu** [comment]
This is an excellent idea, it's time to open the gates now that we've moved on to T6. Let T5 be done quickly to gear up new comers and alts, so we can focus on Hyjal.

**Comment by Creepyy0** [comment]
I have absolutely nothing to say/add. I'm just here to show some support. This is a great idea. (<3 wizs)

**Comment by TotallyNotMyAccount** [comment]
Dew it!

**Comment by MadTrickstar** [comment]
i apprupt dis idea, pls let us skip, JUST DEW IT! DONT LET YOUR DREAMS BE DREAMS!

**Reply by Fustercluk** [reply]
What he said !!!

**Comment by Fustercluk** [comment]
Please let us skip, makes too much sense considering that tier 6 has been released.

**Comment by Cecilius-CC** [comment]
Its good since T5 is already on farm but most of the other bosses are not needed by anyone anymore so pointless to kill them every week.

**Comment by 55Honey** [comment]
It's on the to-do. No ETA can be given momentarily.
Thanks you for your patience.

**Comment by BeefSteaker** [comment]
Closing as this has been applied to live
